src/hyp3_isce2/burst.py

```diff
@@ -6,6 +6,7 @@
 from concurrent.futures import ThreadPoolExecutor
 from dataclasses import dataclass
 from pathlib import Path
+from secrets import token_hex
```

Comment:
Is this the same function we use in hyp3_gamma for creating a hex? It would be good to be consistent